larq/layers_base.py

```diff
@@ -101,17 +101,11 @@ def _is_native_padding(self):
         )
 
     def _get_spatial_padding_same(self, shape):
-        return tuple(
-            _compute_padding(stride, dilation_rate, input_size, filter_size)
-            for stride, dilation_rate, input_size, filter_size in zip(
-                self.strides, self.dilation_rate, shape, self.kernel_size
-            )
-        )
-
-    def _get_spatial_padding_same_shape(self, shape):
         return [
-            size + sum(pad)
-            for size, pad in zip(shape, self._get_spatial_padding_same(shape))
+            _compute_padding(stride, dilation_rate, shape[i], filter_size)
```

Comment:
Unfortunately Tensors don't support iteration in Graph model, so we need to do `shape[i]` instead.